mscore/preferences.cpp

```diff
@@ -92,6 +92,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_APP_STARTUP_FIRSTSTART,                          new BoolPreference(true)},
             {PREF_APP_STARTUP_SESSIONSTART,                        new EnumPreference(QVariant::fromValue(SessionStart::SCORE), false)},
             {PREF_APP_STARTUP_STARTSCORE,                          new StringPreference(":/data/My_First_Score.mscz", false)},
+            {PREF_UI_APP_STARTUP_SHOWTOURS,                        new BoolPreference(true)},
```

Comment:
Could tours be disabled by default in development versions?
We're always hitting the "revert to factory settings" function, so always passing through the tours again and again could be annoying. (plus, I don't believe MuseScore developers really need those tours)
Something like:
```cpp
#if THIS_IS_A_DEVELOPMENT_VERSION
{PREF_UI_APP_STARTUP_SHOWTOURS, new BoolPreference(false)},
#else
{PREF_UI_APP_STARTUP_SHOWTOURS, new BoolPreference(true)},
#endif
```

Reply:
You can disable tours in the startup wizard and on each tour. No more than 2 extra clicks.

Reply:
FWIW, one could express the same concern about the other startup wizard for selecting config options etc.  It's a valid concern, I guess, but also not a big deal - I rather don't like having development buildsdeliberately behave different form release builds.  On the other hand, maybe a command line option to disable the tours could be worthwhile?

Reply:
A command line option to disable the tours? It seems a little bit too specific to me. Maybe, we'll just get over it, and disable the tours after each 'revert to factory settings'.

Reply:
Well, we do have command line options to disable a few other things that kind of sort of seem similar.  But mostly, I think it's just something we should just get over, as you say :-)